src/components/JsxParser.js

```diff
@@ -61,7 +64,7 @@ export default class JsxParser extends React.Component {
     const components = this.props.components.reduce(
       (map, type) => ({
         ...map,
-        [type.prototype.constructor.name.toUpperCase()]: type,
```

Comment:
That's a great point. Would you mind opening an issue for this - and we can address in a separate PR?